pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -997,14 +996,13 @@ def _set_distrib_type_if_training_type_plugin_passed(self):
             self._distrib_type = getattr(self._training_type_plugin, "distributed_backend", None)
 
     def _is_slurm_managing_tasks(self) -> bool:
-        """Returns whether we let SLURM manage the processes or not.
-
-        Returns ``True`` if and only if these conditions match:
+        """Returns whether we let SLURM manage the processes or not. Returns ``True`` if and only if these
+        conditions match:
 
-            - A SLURM cluster is detected
-            - A distributed plugin is being used
-            - The process is not launching in interactive mode
-            - The number of tasks in SLURM matches the requested number of devices and nodes in the Trainer
+        - A SLURM cluster is detected
+        - A distributed plugin is being used
+        - The process is not launching in interactive mode
+        - The number of tasks in SLURM matches the requested number of devices and nodes in the Trainer
```

Comment:
Keep the space before the bullet points?

Reply:
this was modified by the weirdo pre-commit bot

Reply:
The weirdo https://github.com/myint/docformatter project. pre-commit just obeys what we tell it to do.

Reply:
Note that the bot did this in reaction to a change by a human. Probably moving "Returns ``True..." to the first paragraph was the cause.